app/bundles/LeadBundle/Tests/Functional/SearchTestHelper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\LeadBundle\Tests\Functional;
+
+use Mautic\CoreBundle\Test\MauticMysqlTestCase;
+use Mautic\LeadBundle\Entity\Lead;
+use Mautic\LeadBundle\Model\LeadModel;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+abstract class SearchTestHelper extends MauticMysqlTestCase
```

Comment:
```suggestion
abstract class AbstractSearchTest extends MauticMysqlTestCase
```

To maintain consistency, we should adjust the class name. Helper classes should be used as dependencies rather than abstract classes.

There are two options:
Option 1: Rename the class to something like `AbstractSearchTest` and it should be fine
Option 2: Modify it to function as a helper, similar to `app/bundles/EmailBundle/Tests/Functional/Fixtures/EmailFixturesHelper.php`.

Reply:
I'll rename, and if you agree, I think it would also be better for extensibility to move the remaining search-related auxiliary functions in `SearchWithCustomFieldDataFunctionalTest.php` to the abstract.

Reply:
Looks good!